article/app/views/fragments/articleHeaderPaidGarnett.scala.html

```diff
@@ -0,0 +1,32 @@
+@(article: model.Article, page: model.Page, amp: Boolean = false)(implicit request: RequestHeader, context: model.ApplicationContext)
+
+@import views.html.fragments.langAttributes
+@import model.Badges.badgeFor
+@import views.support.Commercial.isPaidContent
+@import views.support.ContributorLinks
+@import views.support.TrailCssClasses.toneClass
+
+<header class="content__head content__head--article">
+    <div class="content__header tonal__header">
+        <div class="gs-container">
+            <div class="content__main-column u-cf">
+                @fragments.meta.metaInline(article)
+                <h1 class="content__headline" articleprop="headline" @langAttributes(article.content)>
+                    @Html(article.trail.headline)
+                </h1>
+            </div>
+        </div>
+    </div>
+    <div class="content__subhead">
+        <div class="gs-container">
+            <div class="content__main-column u-cf">
+                <div class="content__meta-container">
+                    @fragments.commercial.badge(article, page)
+                </div>
```

Comment:
Don't mean to get obsessed with badges, but ought we check `showBadge == true` before adding this element?

Reply:
No worries mate. This is done [here](https://github.com/guardian/frontend/pull/18733/files/c909fba498aee3d282b6603202981a5c8e5808e2#diff-5a1f07b3ac8705d503678b5c34fea95bR45)